root.json

```
@@ -22,7 +22,8 @@
     "Gollum": "gollum.json",
     "HAProxy-Letsencrypt": "haproxy-letsencrypt.json",
     "HandBrake": "handbrake.json",
-    "Headphones": "headphones.json",
+    "HDHomeRun DVR":"hdhomerun_dvr.json",
+	"Headphones": "headphones.json",
```

@Hooverdan96  This looks to have acquired a 'stray' tab char!
```suggestion
    "Headphones": "headphones.json",
```
The file is probably out of kilter with the more common 2 space json in this repo but changing that now would likely upset some merges now. But no tabs if we can help it.